source/includes/mongodb-compatibility-table-ruby.rst

```diff
@@ -0,0 +1,194 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Ruby Driver
+     - MongoDB 8.0
+     - MongoDB 7.0
+     - MongoDB 6.0
+     - MongoDB 5.0
+     - MongoDB 4.4
+     - MongoDB 4.2
+     - MongoDB 4.0
+     - MongoDB 3.6
+     - MongoDB 3.4
+     - MongoDB 3.2
+     - MongoDB 3.0
+     - MongoDB 2.6
```

Comment:
I'll say the same thing here that I said about the old Ruby versions: do we need to keep these ancient server versions here? We've removed support for them from the driver some time ago.

/cc @alexbevi -- curious what you think about this, too.

Reply:
Sounds good, will wait for @alexbevi feedback but again also happy to remove these.

Reply:
Let's remove everything older than MongoDB 4.0 from that matrix.